### DISCUSSION TITLE
connecting to a cluster postgres

### DISCUSSION THREAD

**Discussion by kvatsek** [Flagsmith/flagsmith#2373] [Q&A]
I want to connect Flagsmith to cluster postgres. I add secret 
"DATABASE_URL"    = base64encode("postgres://flagsmith:password@/flagsmith?host=db-psql01.ex.com&host=db-psql02.ex.com&host=db-psql03.ex.com&port=5005&target_session_attrs=primary")

error: core.management.commands.waitfordb WARNING Database not yet ready for connections.
but db is ok

or  I add secret 
"REPLICA_DATABASE_URLS" = base64encode("postgres://flagsmith:password@db-psql01.ex.com:5005/flagsmith,postgres://flagsmith:password@db-psql02.ex.com:5005/flagsmith,postgres://flagsmith:password@db-psql03.ex.com:5005/flagsmith")
but in both cases I get the error
Please tell me how to correctly specify connections to the cluster postgres

error: django.core.exceptions.ImproperlyConfigured: settings.DATABASES is improperly configured. Please supply the ENGINE value.

**Comment by dabeeeenster** [comment]
Why are you trying to base64 encode `DATABASE_URL`?

**Reply by kvatsek** [reply]
So tuned the secrets in kubernetes.
finally DATABASE_URL and REPLICA_DATABASE_URLS are obtained without encoding
